articles/protocols/saml/samlp.md

```diff
@@ -29,7 +29,15 @@ These are the parameters used to configure a SAML Identity Provider:
 You need to replace the `ACCESS_TOKEN` header value, with a Management APIv2 Token. For information on how to do that refer to [The Auth0 Management APIv2 Token](/api/management/v2/tokens).
 :::
 
-* The __SAML Request Binding__ (also called the __Protocol Binding__): sent to the IdP from Auth0, set as either `HTTP-Redirect` or `HTTP-Post`
+* The __SAML Request Binding__ (also called the __Protocol Binding__): sent to the IdP from Auth0. If possible, dynamically set the value based on `connection.options.protocolBinding`:
+
+| `connection.options.protocolBinding` value         | SAML Request Binding value |
+| -------------------------------------------------- | -------------------------- |
+| `` or not present                                  | HTTP-Redirect              |
+| urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect | HTTP-Redirect              |
+| urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST     | HTTP-POST                  |
+
+    If dynamically setting the value isn't possible, then set as either `HTTP-Redirect` (default) or `HTTP-Post` if you selected this option in **Protocol Binding**
```

Comment:
The matrix is OK now but the text underneath is not displayed correctly:

![image](https://user-images.githubusercontent.com/11715799/28597664-d68282f6-7164-11e7-9938-a927cac1b5b4.png)